### DISCUSSION TITLE
how to test policy as code post deployment?

### DISCUSSION THREAD

**Discussion by tmitra1990** [accurics/terrascan#1078] [Q&A] [answered]
How to test policies are applied in the azure resources properly post deployment?

**Comment by cesar-rodriguez** [comment, marked as answer]
Hi @tmitra1990,

Sorry for the late reply as I just learned about the discussions feature within GitHub. Terrascan is mostly a static code analysis tool for code itself. You can use it to apply security controls within your code repository, pipelines, locally on your desktop, etc. It doesn't have the capability to inspect your cloud environment to ensure the same policies are applied.
